code/game/objects/items/devices/scanners.dm

```diff
@@ -273,52 +297,51 @@ SLIME SCANNER
 		else
 			msgs += "<span class='info'>Blood level [blood_percent] %, [blood_volume] cl, type: [blood_type]</span>"
 
+	// Body temperature.
 	msgs += "<span class='notice'>Body Temperature: [round(H.bodytemperature-T0C, 0.01)]&deg;C ([round(H.bodytemperature*1.8-459.67, 0.01)]&deg;F)</span>"
 	msgs += "<span class='notice'>Subject's pulse: <font color='[H.pulse == PULSE_THREADY || H.pulse == PULSE_NONE ? "red" : "blue"]'>[H.get_pulse()] bpm.</font></span>"
 
+	// Implants.
 	var/implant_detect
 	for(var/obj/item/organ/internal/O in H.internal_organs)
 		if(O.is_robotic() && !O.stealth_level)
-			implant_detect += "[H.name] is modified with a [O.name].<br>"
+			implant_detect += "[O.name].<br>"
 	if(implant_detect)
 		msgs += "<span class='notice'>Detected cybernetic modifications:</span>"
 		msgs += "<span class='notice'>[implant_detect]</span>"
+
+	// Clone damage.
 	if(H.gene_stability < 40)
 		msgs += "<span class='userdanger'>Subject's genes are quickly breaking down!</span>"
 	else if(H.gene_stability < 70)
 		msgs += "<span class='danger'>Subject's genes are showing signs of spontaneous breakdown.</span>"
 	else if(H.gene_stability < 85)
 		msgs += "<span class='warning'>Subject's genes are showing minor signs of instability.</span>"
 
+	// Husk.
```

Comment:
"Don’t over-comment. Too many comments can make code harder to read and understand. Only add comments when necessary, not to every line of code. Avoid redundant comments."